superset-frontend/src/profile/components/UserInfo.tsx

```diff
@@ -19,17 +19,36 @@
 import React from 'react';
 import Gravatar from 'react-gravatar';
 import moment from 'moment';
-import { Panel } from 'react-bootstrap';
-import { t } from '@superset-ui/core';
+import Collapse from 'src/common/components/Collapse';
+import { t, styled } from '@superset-ui/core';
 import { UserWithPermissionsAndRoles } from '../../types/bootstrapTypes';
 
 interface UserInfoProps {
   user: UserWithPermissionsAndRoles;
 }
 
+const StyledContainer = styled.div`
```

Comment:
This is failing the "shows the right user-id" unit test. It's saying "TypeError: Cannot read property 'grayscale' of undefined" which probably means the test needs to replace the standard `mount` with `import { styledMount as mount } from 'spec/helpers/theming';`